doc/classes/Viewport.xml

```diff
@@ -165,6 +166,7 @@
 			<return type="void" />
 			<param index="0" name="text" type="String" />
 			<description>
+				Calls a [code]set_text()[/code] method on the currently focused [Control].
```

Comment:
```suggestion
				Calls the [code]set_text()[/code] method on the currently focused [Control].
```

Reply:
BTW, the code just seems to call `set_text` without caring whether the function is defined. Doesn't it raise errors when the selected control is one without `set_text` method?

Reply:
No, the C++ `call()` will fail silently and instead *return* an error.